platform/core/PolyPodCoreAndroid/core/src/main/java/coop/polypoly/core/CoreExceptions.kt

```diff
@@ -0,0 +1,30 @@
+package coop.polypoly.core
+
+import Failure
+
+@ExperimentalUnsignedTypes
```

Comment:
`failure.code` may be unsigned? I don't see any other possible instance here.

Reply:
Funny enough, `failure.code` is `Byte`,  not `UByte`, so that doesn't seem to be it. But I guess we're accessing _some_ unsigned type below, otherwise this wouldn't be needed (and we'd get a warning I think).

Reply:
Here `Failure` is used which is annotated with `ExperimentalUnsignedTypes`. The FlatBuffers model body can contain internal unsigned types. `Failure` itself does not have any, but Flatbuffers generator automatically adds the annotation.